src/scripts/check-links.js

```diff
@@ -14,7 +14,7 @@ function checkLinks(args) {
   var argv = minimist(process.argv.slice(2));
   var skip = argv.skip || 0;
 
-  var tap = parser();
+  var tap = new Parser();
```

Comment:
Do these changes fix something or just coding style preference? It feels a bit odd to capitalize `Parser` here when none of the other imports are capitalized.

Reply:
It has become so since the `tap-parser` is transformed into a class from version `5.4.0` (https://github.com/tapjs/tap-parser/compare/v5.4.0...master#diff-a1e0ee62c91705696ddb71aa30ad4f95L13)